legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -1,7 +1,30 @@
 <div class="business-details-container">
    <table class="business-details-table" role="presentation">
       <tr>
-         {% if header.reportType == 'noticeOfArticles' %}
+         {% if reportType == 'businessSummary' %}
```

Comment:
should this be `header.reportType` as opposed to `reportType`?

Reply:
nvm, i see where this comes from(`BusinessSummary._get_template_data` function) now.